recipes/micro/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "micro" %}
+{% set version = "2.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/zyedidia/micro.git
+  git_rev: v{{ version }}
+  # url: https://github.com/zyedidia/micro/archive/refs/tags/v{{ version }}.tar.gz
+  # sha256: 3e1b7483aa2afffa8afc2c710e22541961ac3de5e026efb4bdddc8ade00d4de0
+
+build:
+  skip: true  # [osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }} >=1.15
+    - go-licenses
+    - make
+
+test:
+  commands:
+    - micro --version
+
+about:
+  home: https://micro-editor.github.io/
+  license: MIT
+  license_file:
+    - LICENSE
+    - "{{ RECIPE_DIR }}/thirdparty-licenses/"
```

Comment:
Haven't been able to get this functionality to work - specifying a folder here always results in copy errors:
```
Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/site-packages/conda_build/utils.py", line 537, in _copy_with_shell_fallback
    subprocess.check_call('cp -a {} {}'.format(src, dst), shell=True,
  File "/opt/conda/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'cp -a /home/conda/staged-recipes-copy/recipes/micro/thirdparty-licenses/layeh.com/gopher-luar/array_test.go /home/conda/staged-recipes/build_artifacts/micro_1618367372622/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/info/licenses/LICENSE0/layeh.com/gopher-luar/array_test.go' returned non-zero exit status 1.
```